tests/test_ddk.py

```diff
@@ -105,26 +124,26 @@ def test_sini_from_value(self):
             self.modelJ1713.validate()
 
     def test_sini_from_par(self):
-        temp_par_str = """PSR  J1713+0747
-               LAMBDA 256.66  1 0.001
-               BETA 30.70036  1 0.001
-               PMLAMBDA 5.2671  1  0.0021
-               PMBETA  -3.4428  1  0.0043
-               PX  0.8211  1  0.0258
-               F0  218.81  1  0.01
-               PEPOCH  55391.0
-               BINARY  DDK
-               A1 32.34  1  0.001
-               E  0.074  1  0.001
-               T0 55388.836  1  0.0002
-               PB 67.825129  1  0.0001
-               OM 176.19845  1  0.0013
-               M2  0.283395  1  0.0104
-               KOM   83.100  1  1.800
-               SINI  0.8     1  0.562
-               K96  1"""
+        test_par_str = self.temp_par_str + "\n SINI  0.8     1  0.562"
```

Comment:
Why is this supposed to raise a ValueError? Can you enhance the exception check so that the test is clear?

Reply:
I can't tell what the error actually is - maybe something about having SINI? Above that seems to produce only warnings in `validate()`?

Reply:
I believe this is changed to a TimingModelError.

Reply:
Right, but what is actually wrong with the model? Why do we expect this to raise an exception? That should be clear from the test code.

Reply:
We don't have a good translator from SINI--> KIN. With SINI, the code will get confused.

Reply:
Okay, thanks. Please make that clear in the test code, perhaps by verifying that the exception raised includes this information.